src/app/home-page/recent-events/recent-events.component.html

```diff
@@ -7,9 +7,17 @@
       fxLayoutAlign="space-between center"
     >
       <div fxLayout>
-        <a [routerLink]="'/users/' + event.initiatorUser?.username">
-          <img src="{{ event?.initiatorUser?.avatarUrl }}" alt="User avatar" />
-        </a>
+        <div *ngIf="event?.initiatorUser?.username; else defaultAvatar">
+          <a
+            *ngIf="event.type !== EventType.APPROVEORG && event.type !== EventType.REJECTORG && event.type"
+            [routerLink]="'/users/' + event.initiatorUser?.username"
```

Comment:
The question mark in `initiatorUser?` is redundant -- you already check for it on line 10.